gspread/worksheet.py

```diff
@@ -3201,3 +3204,86 @@ def cut_range(
         }
 
         return self.client.batch_update(self.spreadsheet_id, body)
+
+    def add_validation(
+        self,
+        source: str,
+        condition_type: ValidationConditionType,
+        *values: Iterable[Any],
+        inputMessage: str | None = None,
```

Comment:
This syntax won't work on earliest versions of python we support. please use `Optional[str]` instead